packages/dso-toolkit/components/Voorbeelden/modal-en-progress-indicator.njk

```diff
@@ -0,0 +1,18 @@
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
+<body {{ className('dso-modal-open', [noMaxHeight, 'dso-scripted-height']) }}>
+<div class="dso-modal" tabindex="-1" role="dialog" aria-labelledby="modal-dialog">
```

Comment:
`aria-labelledby="modal-dialog"` -> er bestaat op deze pagina geen element (heading) met dat id. Dus daarmee klopt eigenlijk deze markup niet...

Reply:
Vergelijk met:
![image](https://user-images.githubusercontent.com/2689559/111342477-c853cf80-867a-11eb-97ed-0bb593115f39.png)
https://dso-toolkit.nl/21.1.0/components/detail/modal.html

De vraag is nu natuurlijk, waar zouden we dan met `aria-labelledby` naar moeten verwijzen. Maar daarmee komt eigenlijk ook direct de vraag op, hoe toegankelijk eigenlijk ueberhaupt een progress indicator in een modal is. Misschien iets om even met @timveld over te sparren?